Bloon/Features/Intruder Backend/Agents/Commands/Stats.cs

```diff
@@ -12,6 +12,7 @@ namespace Bloon.Commands
     using DSharpPlus.CommandsNext.Attributes;
     using DSharpPlus.Entities;
 
+    [StatsAttribute]
```

Comment:
Attribute could and should be moved to the feature.

Reply:
attribute is used in more than one location tho for the features.

Reply:
Are there more stats commands that are going to use this?

Reply:
yes

Reply:
Okily dokily